tix_clock/tix_clock.ino

```diff
@@ -9,13 +9,38 @@ const int CLOCK_PIN = 4;
 const int NUMBER_OF_OUTPUT_PINS = 3;
 int OUTPUT_PINS[] = {DATA_PIN, LATCH_PIN, CLOCK_PIN};
 const int SECONDS_PERIOD_IN_MS = 1000;
+const int SQUARE_MATRIX_SIZE = 8;
+const int hourTensNumberOfPossibleLeds = 2;
+const int hourUnitsNumberOfPossibleLeds = 9;
 int hour = 0;
 int minute = 0;
 int second = 0;
 int hourTens = 0;
 int hourUnits = 0;
 int minuteTens = 0;
 int minuteUnits = 0;
+int tixClock[SQUARE_MATRIX_SIZE][SQUARE_MATRIX_SIZE] = {
+                                           {0,0,0,0,0,0,0,0},
+                                           {0,0,0,0,0,0,0,0},
+                                           {0,0,0,0,0,0,0,0},
+                                           {0,0,0,0,0,0,0,0},
+                                           {0,0,0,0,0,0,0,0},
+                                           {0,0,0,0,0,0,0,0},
+                                           {0,0,0,0,0,0,0,0},
+                                           {0,0,0,0,0,0,0,0},
+};
+int hourTensPossibleleds[hourTensNumberOfPossibleLeds][2] = {{0,1},
+                                                            {1,0},                                                         
+};
+int hourUnitsPossibleleds[hourUnitsNumberOfPossibleLeds][2] = {{0,4},
+                                                               {0,5},
+                                                               {1,4},
```

Comment:
There's one coordinate missing. Please add it .

Reply:
I just added the missing coordinate